README.md

```diff
@@ -753,6 +753,30 @@ fetchContactsByPhone: (phoneNumbers: Array<string>) => Promise<Array<{
 }>>;
 ```
 
+### addOrEditContact
+
+Opens native UI to add or edit a contact in the device's phonebook.
+
+```ts
+addOrEditContact: (phoneNumber: string) => Promise<{
+    phoneNumber: string;
+    firstName?: string;
+    middleName?: string;
+    lastName?: string;
+    encodedAvatar?: string;
+}>
+```
+
+-   If phoneNumber already exists in the device phonebook, the user will be able
+    to edit the information.
+-   If phoneNumber is saved under multiple names in the phonebook, when editing
+    it should choose the first one alphabetically
+-   If phoneNumber doesn't exist in the phonebook, the user will be able to add
+    it, providing the related info.
+
+Once the user has added or updated the contact, native returns the new
+information (all last values of every property).
```

Comment:
What happens if the user cancels the creation or the edition?

Reply:
Good question. I'd say that the caller doesn't need to know that detail, I would just return the information of the contact (only the msisdn if it was a number not existing in the phonebook). What do you think?